subprocess.h

```diff
@@ -118,14 +118,17 @@ subprocess_weak int subprocess_create(const char *const command_line[],
 /// for a child process (each element of the form FOO=BAR). The last element
 /// must be NULL to signify the end of the array.
 /// @param out_process The newly created process.
+/// @param process_cwd The CWD of the newly created process. If null, will be 
```

Comment:
can you change `CWD` -> `current working directory` just so that people don't have to google (I generally dislike acronyms!).